### DISCUSSION TITLE
Interoperability with radicle

### DISCUSSION THREAD

**Discussion by hoijui** [git-bug/git-bug#1130] [Q&A] [answered]
[Radicle](https://radicle.xyz/) is a P2P, OpenSource git hosting system (Though there is also a crypo-currency with the same name -> different project).
Besides the code repo, it also stores issues.

I was wondering, what would be the best way to get git-bug and radicle to work together: create a bridge? try to unite the issues storage formats? ...

**NOTE**
This is not at all a pressing issue, I am just curious. feel free to ignore/postpone this.

**Comment by sudoforge** [comment, marked as answer]
based on the [user guide for working with issues in radicle](https://radicle.xyz/guides/user#working-with-issues), and their [COB format](https://radicle.xyz/guides/protocol/#collaborative-objects), yes, to "integrate" `git-bug` with `radicle`, one would build a bridge to import/export from/to `radicle`, converting the COB to a git-bug issue.
